cli/src/main.rs

```diff
@@ -262,7 +263,11 @@ fn main() -> Result<()> {
     let stdout_subscriber = tracing_subscriber::fmt::layer()
         .compact()
         .without_time()
-        .with_filter(EnvFilter::from_default_env());
+        .with_filter(
+            EnvFilter::builder()
+                .with_default_directive(LevelFilter::OFF.into())
+                .from_env_lossy(),
```

Comment:
What's the reason for this change?

Reply:
This was the behavior we had before the introduction of the tracing crate. We sometimes log an error in the info command. While it is errors that are fatal for the communication normally, it's no issue in the info command as we try on. But we don't want to see those errors in standar output.